.changesets/config_subgraph_include_errors allow_and_deny_list.md

```diff
@@ -0,0 +1,29 @@
+### `include_subgraph_errors` fine grained control ([Issue #6402](https://github.com/apollographql/router/pull/6402)
+
+Update `include_subgraph_errors` with additional configuration options for both global and subgraph levels. This update provides finer control over error messages and extension keys for each subgraph. 
+For more details, please read the [subgraph-error-inclusion](https://www.apollographql.com/docs/apollo-server/observability/subgraph-error-inclusion/).
```

Comment:
The link is to Apollo Server right now, but [this](https://www.apollographql.com/docs/graphos/routing/observability/subgraph-error-inclusion) looks like the link you want:

```suggestion
For more details, please read [subgraph error inclusion](https://www.apollographql.com/docs/graphos/routing/observability/subgraph-error-inclusion).
```